sphinxcontrib/confluencebuilder/builder.py

```diff
@@ -181,6 +190,8 @@ def prepare_writing(self, docnames):
             if not doctitle:
                 continue
 
+            first_section.remove(first_section[idx])
```

Comment:
I assume this change is to remove the title from the page's content and solely use Confluence's title?

I would recommend that this change be in it's own commit and is configurable (I wouldn't mind it even being defaulted on). I still think there can be value added by some individuals to maintain both title values (ex. when using configurable prefixes such as `confluence_publish_prefix`).